docs/kube-api-custom-ocp-versions.md

```diff
@@ -0,0 +1,94 @@
+# Custom OpenShift Versions (Dynamic z-stream Support)
```

Comment:
nit: I don't think this intends to discuss "custom versions". Red Hat makes releases and decides what the versions are, and here we are just helping the user select versions. I would refer to this as "Selecting OpenShift Versions", "Choosing OpenShift Versions", or similar.

Likewise, the fact that the API is "dynamic" does not mean much to the user. They're using a k8s API, so it's assumed that API use should not require restarting a service.